int32 tag_id         # ID of the detected AprilTag
float32 x            # Relative x position of the marker in the camera frame
float32 y            # Relative y position of the marker in the camera frame
float32 z            # Relative z position of the marker in the camera frame
float32 angle        # Relative angle (yaw) of the marker in radians